### PR TITLE
Drupal9.1 compatibility

### PR DESCRIPTION
**Pull request by ccjjmartin**
## Purpose:
- Drupal9.1 compatibility

## Testing:
- Pull down this branch
- If you are using an existing clone of the project check your settings.php file for `$databases['default']['default']` and if necessary update the lando config from referencing drupal8 to drupal9.
- `lando destroy -y` # Destroy any existing lando instances
- `lando start` # Upon build we now do a composer install, this tests the out of the box experience
- Verify that composer installs without error
- Verify that you can install a new database without errors
- Verify that you can set Emulsify as the default theme without errors

### CODE REVIEW

**Comment by ccjjmartin** [comment]
@americkson This is the associated branch on Drupal.org: https://www.drupal.org/project/sous/issues/3191079#comment-13956075

File: .lando.yml [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
-recipe: drupal8
+recipe: drupal9
 config:
   webroot: web
-  drush: 9
-  php: '7.2'
+  php: '7.3'
+services:
+  appserver:
+    run:
```
Comment (americkson) [inline .lando.yml:8]: Same as the 8.x version. I think we might want to keep the composer install out of the lando config.

Comment (americkson) [reply .lando.yml:8]: Answered here: https://github.com/fourkitchens/sous-drupal-project/pull/11#discussion_r552126119

**Review by americkson — APPROVED** [review]

**Comment by ccjjmartin** [comment]
Same story here, closing since the code is already on the right branch, I am going to update the default branch on both to 2.x @americkson

**Pull request closed without merging**